src/vinylcontrol/defs_vinylcontrol.h

```diff
@@ -8,22 +8,38 @@ constexpr int VINYL_STATUS_OK = 1;
 constexpr int VINYL_STATUS_WARNING = 2;
 constexpr int VINYL_STATUS_ERROR = 3;
 
-#define MIXXX_VINYL_FINALSCRATCH "Final Scratch (crappy)"  // Not currently used
-#define MIXXX_VINYL_MIXVIBESDVS "MixVibes DVS V2 Vinyl"
 #define MIXXX_VINYL_SERATOCV02VINYLSIDEA "Serato CV02 Vinyl, Side A"
 #define MIXXX_VINYL_SERATOCV02VINYLSIDEB "Serato CV02 Vinyl, Side B"
 #define MIXXX_VINYL_SERATOCD "Serato CD"
 #define MIXXX_VINYL_TRAKTORSCRATCHSIDEA "Traktor Scratch MK1 Vinyl, Side A"
 #define MIXXX_VINYL_TRAKTORSCRATCHSIDEB "Traktor Scratch MK1 Vinyl, Side B"
+#define MIXXX_VINYL_MIXVIBESDVS "MixVibes DVS V2 Vinyl"
+#define MIXXX_VINYL_MIXVIBES7INCH "MixVibes 7 inch"
+#define MIXXX_VINYL_PIONEERA "Pioneer RekordBox DVS Control Vinyl, Side A"
+#define MIXXX_VINYL_PIONEERB "Pioneer RekordBox DVS Control Vinyl, Side B"
+
+#define MIXXX_VINYL_SERATOCV02VINYLSIDEA_XWAX_NAME "serato_2a"
+#define MIXXX_VINYL_SERATOCV02VINYLSIDEB_XWAX_NAME "serato_2b"
+#define MIXXX_VINYL_SERATOCD_XWAX_NAME "serato_cd"
+#define MIXXX_VINYL_TRAKTORSCRATCHSIDEA_XWAX_NAME "traktor_a"
+#define MIXXX_VINYL_TRAKTORSCRATCHSIDEB_XWAX_NAME "traktor_b"
+#define MIXXX_VINYL_MIXVIBESDVS_XWAX_NAME "mixvibes_v2"
+#define MIXXX_VINYL_MIXVIBES7INCH_XWAX_NAME "mixvibes_7inch"
+#define MIXXX_VINYL_PIONEERA_XWAX_NAME "pioneer_a"
+#define MIXXX_VINYL_PIONEERB_XWAX_NAME "pioneer_b"
+
+#define MIXXX_VINYL_DEFAULT_XWAX_NAME MIXXX_VINYL_SERATOCV02VINYLSIDEA_XWAX_NAME
 
 // Default recommended leadin times for various vinyl types.
-#define MIXXX_VINYL_FINALSCRATCH_LEADIN 0
-#define MIXXX_VINYL_MIXVIBESDVS_LEADIN 0
 #define MIXXX_VINYL_SERATOCV02VINYLSIDEA_LEADIN 28
 #define MIXXX_VINYL_SERATOCV02VINYLSIDEB_LEADIN 28
 #define MIXXX_VINYL_SERATOCD_LEADIN 0
 #define MIXXX_VINYL_TRAKTORSCRATCHSIDEA_LEADIN 10
 #define MIXXX_VINYL_TRAKTORSCRATCHSIDEB_LEADIN 10
+#define MIXXX_VINYL_MIXVIBESDVS_LEADIN 0
+#define MIXXX_VINYL_MIXVIBES7INCH_LEADIN 0
+#define MIXXX_VINYL_PIONEERA_LEADIN 0
```

Comment:
pioneer is modeled on the serato vinyl, so let's set the leadin to 10

Reply:
OK.